source/user-manual/reference/tools/keystore-tool.rst

```diff
@@ -0,0 +1,35 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: Tool where sensitive configuration data, including user credentials for the indexer, can be securely stored. 
```

Comment:
IMHO indexer should be removed

Reply:
Done in [Make requested changes](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/166a6616aa43bcfd53cffd26dd40a3764a21f901) commit